lang/python/python-dns/Makefile

```diff
@@ -20,31 +20,22 @@ PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_CPE_ID:=cpe:/a:debian:python-dns
 
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-dnspython-$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/dnspython-$(PKG_VERSION)
```

Comment:
curios: can we then just remove `PKG_BUILD_DIR`?

Reply:
I tried :joy: the default `PKG_BUILD_DIR` is `python-dns-$(PKG_VERSION)` (`PKG_NAME` is python-dns) but the tarball is `dnspython-$(PKG_VERSION)`.

Reply:
oh ok;